src/components/MyAccount/CheckoutsTab.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Box,
+  Button,
+  Icon,
+  Link,
+  Table,
+  Text,
+  useModal,
+} from "@nypl/design-system-react-components"
+import type { Checkout, Patron } from "../../types/accountTypes"
+import { useEffect, useState } from "react"
+import { BASE_URL } from "../../config/constants"
+import { getTitle } from "../../utils/myAccountUtils"
+import styles from "../../../styles/components/MyAccount.module.scss"
+
+const CheckoutsTab = ({
+  checkouts,
+  patron,
+}: {
+  checkouts: Checkout[]
+  patron: Patron
+}) => {
+  const checkoutsHeaders = [
+    "Title",
+    "Barcode",
+    "Call number",
+    "Due back by",
+    "Manage checkout",
+  ]
+
+  function RenewButton(checkout, patron) {
+    const [isButtonDisabled, setButtonDisabled] = useState(false)
+    const { onOpen, Modal } = useModal()
+    const [modalProps, setModalProps] = useState(null)
+    const successModalProps = {
+      bodyContent: (
+        <Box className={styles.modalBody}>
+          <Text sx={{ marginLeft: "l" }}>
+            Your item was renewed. It is now due back on {checkout.date}.
+          </Text>
+        </Box>
+      ),
+      closeButtonLabel: "OK",
+      headingText: (
+        <Box className={styles.modalHeading}>
+          <Icon
+            size="large"
+            name="actionCheckCircleFilled"
+            color="ui.success.primary"
+          />
+          <Text sx={{ marginBottom: "0px" }}> Renewal successful </Text>
+        </Box>
+      ),
+    }
+    const failureModalProps = {
+      bodyContent: (
+        <Box className={styles.modalBody}>
+          <Text sx={{ marginLeft: "l", marginRight: "m" }}>
+            We were unable to renew your item. Please try again or{" "}
+            <Link href="https://www.nypl.org/get-help/contact-us">
+              contact us
+            </Link>{" "}
+            for assistance.
+          </Text>
+        </Box>
+      ),
+      closeButtonLabel: "OK",
+      headingText: (
+        <Box className={styles.modalHeading}>
```

Comment:
The heading is not displaying.
<img width="589" alt="Screen Shot 2024-03-08 at 10 29 34 AM" src="https://github.com/NYPL/research-catalog/assets/1280564/55e6256a-1bfc-4d42-91da-953f8a274699">

This might be too restrictive but we expect an `<h*>` HTML element or a DS `Heading` component. Replace `Box` with `<Heading ...><>...</></Heading>`. Some styles will need an update:
<img width="565" alt="Screen Shot 2024-03-08 at 10 49 28 AM" src="https://github.com/NYPL/research-catalog/assets/1280564/30e0fa2a-3029-461a-8438-315f9e1c361d">

Reply:
For sure but this is not happening locally for me ? Why would that be

Reply:
I'll test again. Maybe I did not run `npm install` to get the updated ds version?